posthog/api/action.py

```diff
@@ -330,10 +331,10 @@ def _stickiness(self, filtered_events: QuerySet, entity: Entity, filter: Filter)
             'count': sum(data)
         }
 
-    def _breakdown_label(self, entity: Entity, value: str) -> Dict[str, Optional[str]]:
+    def _breakdown_label(self, entity: Entity, value: Any) -> Dict[str, Optional[str]]:
```

Comment:
Any tends to be an antipattern. Is it actually `Union[str, int]`?

Reply:
yup, forgot about Union. thanks